src/style.css

```diff
@@ -0,0 +1,239 @@
+html {
+  -ms-overflow-style: none; /* IE and Edge */
+  scrollbar-width: none; /* Firefox */
+}
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+html::-webkit-scrollbar {
+  display: none;
+}
+
+/* Hide scrollbar for IE, Edge and Firefox */
+
+body {
+  display: block;
+  overflow-x: hidden;
+}
+
+.flex {
+  display: flex;
+}
+
+.none {
+  display: none;
+}
+
+nav {
+  width: 100%;
+  align-items: center;
+  justify-content: space-around;
+  font-weight: bold;
+  margin-left: 4%;
+}
+
+header {
+  background-color: #040f0f;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #e8e8e9;
+}
+
+.nav-link:hover {
+  color: #289944;
+}
+
+.logo {
+  font-family: "Allison", cursive;
+  font-size: 30px;
+}
+
+ul {
+  list-style: none;
+  justify-content: space-around;
+  width: 70%;
+  margin: auto;
+  font-size: 20px;
+  font-family: "Kaisei Decol", serif;
+}
+
+footer {
+  width: 100%;
+  border-top: 1px solid black;
+  background-color: #040f0f;
+  color: #e8e8e9;
+  font-family: "Kaisei Decol", serif;
+}
+
+.footer {
+  padding: 2%;
+  font-size: 20px;
+}
+
+/* Home */
+.category {
+  flex-wrap: wrap;
+  width: 85%;
+  margin: auto;
+  padding: 10% 0 0 0;
+  justify-content: space-around;
+}
+
+.sec {
+  flex-direction: column;
+  padding-top: 5%;
+}
+
+.sec h4 {
+  width: 50%;
+  margin: auto;
+  text-align: center;
+}
+
+.card {
+  flex-direction: column;
+  width: 30%;
+  border: solid;
+  border-width: 1px;
+  margin: 0 0 10% 0;
+  color: #0e3013;
+  font-family: "Kaisei Decol", serif;
+}
+
+.card img {
+  width: 90%;
+  height: 60%;
+  margin: auto;
+  border-radius: 2%;
+}
+
+.name-like {
+  width: 90%;
+  justify-content: space-between;
+  margin: 10px 15px 0 15px;
+}
+
+.like {
+  background-color: white;
+  border: none;
+  cursor: pointer;
+}
+
+button {
+  font-family: "Kaisei Decol", serif;
+}
+
+.comments-button {
+  width: 70%;
+  margin: 0 auto 3% auto;
+  background-color: #0e3013;
+  color: #e8e8e9;
+  font-weight: bold;
+}
+
+.btn:hover {
+  color: #289944;
+}
+
+.popUp {
+  display: none;
+  grid-template-rows: auto;
+  grid-template-columns: auto;
+  width: 50%;
+  text-align: center;
+  border: 1.5px solid black;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #e8e8e9;
+  background: linear-gradient(360deg, rgba(0, 128, 49, 0.897), #0e3013ea);
+  height: 90%;
+  overflow-y: auto;
+
+  /* overflow: -moz-scrollbars-none; */
+  padding: 0.5rem 0;
+  font-family: "Kaisei Decol", serif;
+  font-size: 1rem;
+  z-index: 1;
+  border-radius: 5%;
```

Comment:
- [ ] In the popUp window the image does not have the proper size. Kindly change the image height and you can make the overflow property of this `.popUp` class to scroll. 
![image](https://user-images.githubusercontent.com/64894927/132676356-ab6639a3-7087-4012-9525-23a5107bccc8.png)
![image](https://user-images.githubusercontent.com/64894927/132676400-e8818824-8ffc-4464-9433-ea26bffd7089.png)

Reply:
I fixed the image size. The scroll property was already set only the bar was hidden for estetics. You can scroll using the keyboard arrows.